Sources/Recycling/ListViews/Wishlist/Cell/WishListViewCell.swift

```diff
@@ -0,0 +1,221 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public protocol WishListViewCellDataSource: class {
+    func wishListViewCell(_ wishlistListViewCell: WishListViewCell, loadImageForModel model: WishListViewModel, imageWidth: CGFloat, completion: @escaping ((UIImage?) -> Void))
+    func wishListViewCell(_ wishlistListViewCell: WishListViewCell, cancelLoadingImageForModel model: WishListViewModel, imageWidth: CGFloat)
+}
+
+public class WishListViewCell: UITableViewCell {
+    // MARK: - External properties
+    
+    /// The loading color is used to fill the image view while we load the image.
+    public var loadingColor: UIColor?
+    
+    /// A data source for the loading of the image
+    public weak var dataSource: WishListViewCellDataSource?
+    
+    // MARK: - Internal properties
+    
+    private static let layerOpacity: Float = 0.85
+    private static let cornerRadius: CGFloat = 8
+    
+    private lazy var adImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = WishListViewCell.cornerRadius
+        imageView.contentMode = .scaleAspectFill
+        return imageView
+    }()
+    
+    private lazy var leftImageDetail: Label = {
+        let label = Label(style: .title4)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.backgroundColor = .licorice
+        label.textColor = .milk
+        label.textAlignment = .center
+        label.layer.opacity = WishListViewCell.layerOpacity
+        return label
+    }()
+    
+    private lazy var rightImageDetail: Label = {
```

Comment:
This should be a Ribbon instead. https://github.com/finn-no/FinniversKit/tree/master/Sources/Components/Ribbon